### ISSUE TITLE
[BUG] Steam overlay and status not functioning for EA Desktop and UWP parsers (Windows OS)

### ISSUE THREAD

**Issue by TheWayTheTruthTheLife** [doZennn/steam-rom-manager#694]
- **Describe the bug**

Games added to the Steam library using EA Desktop and UWP parsers do not enable the steam overlay when the game is running.

The games launch properly through their own launchers when they are started through Steam, but Steam immediately drops the "now playing" status and the Steam overlay does not work.

Tested all of the following EA Desktop games, and they all experienced this issue:
> Dragon Age: Inquisition
> Mass Effect: Andromeda
> Star Wars: Battlefront II - Celebration Edition (Linked through Epic Games)

Tested the following UWP game, and it experienced this issue:
> Minecraft Launcher (the one that includes all editions of the game)

The parsers experiencing this issue include:
```
EA Desktop
UWP
```

- **Steps to reproduce the behavior**

1. Have games installed using EA Desktop and UWP.
2. Configure EA Desktop and UWP parsers and save the games to the Steam library using SRM, and they appear in the Steam app.
3. Launch an EA Desktop or UWP game using Steam.
4. Steam briefly updates the "now playing" status to your game, and then immediately resets to no game.
5. If the game was EA, the EA launcher opens properly and the game runs. If the game was UWP, the UWP game opens.
6. **Press Shift + Tab to open the Steam overlay (or your custom keybind) and nothing happens. The overlay didn't work, and Steam doesn't even know you're running the game.**
7. If the game is EA, press the keybind for EA's overlay. EA's own overlay works properly even though Steam does not.

- **Expected behavior**

Every other parser I've used opens the Steam overlay properly.

The parsers I've used that open the Steam overlay properly are:

> Amazon Games
> Battle.net
> Epic
> GOG Galaxy (for everything except old games, like Star Wars Battlefront II 2005)
> Non-SRM Artwork

The EA Desktop and UWP parsers should open the Steam overlay and show the "now playing" status in Steam, but they do not.

**Comment by cbartondock** [comment]
I don't think this is something SRM can fix, as it has more to do with how those launchers interact with the graphics card API / order of launch. I will look into it.

**Comment by TheWayTheTruthTheLife** [comment]
> I don't think this is something SRM can fix, as it has more to do with how those launchers interact with the graphics card API / order of launch. I will look into it.

Thank you, appreciate it.

The new steam recording feature leverages the steam overlay to work. So if there is a solution to this, it would allow steam to be used as a game recording, streaming, and screenshot/video hosting profile for games across all launchers, which is really cool.

**Comment by moi952** [comment]
Hello, @cbartondock  do you have any news regarding this problem?
I have the same problem but worse, the game launches but behind steam so I have to take out a keyboard to press windows and click on the game in the taskbar to make it appear on the screen.

Thank you very much for this really great tool 😀

**Comment by kencinder** [comment]
Closing this, it's not only stale but SRM has never had anything to do with whether the overlay works with a non steam app or not anyway.
